libtiff/libtiff_ctypes.py

```diff
@@ -338,23 +343,74 @@ def extender_pyfunc(tiff_struct):
 
 
 def add_tags(tag_list):
```

Comment:
I agree this function name could be better. On the other hand, for backward compatibility, we'd need to keep that old name for while (in addition to the new name). Let's do that in a separate PR.
One thing which would be handy too is to keep a reference to the TIFFExtender returned, because for now, if the caller drops the reference to that object, libtiff crashes, which brings the whole process down. Not very user friendly.